File: algorithm/70. Climbing Stairs/70.cpp
class Solution {
public:
    int climbStairs(int n) {
        if(n<2) return 1;
        vector<int> a(n,1);
        a[1]=2;
        for(int i=2; i<n; i++)
        {
            a[i]=a[i-1]+a[i-2];
        }
        return a[n-1];
    }
};

File: algorithm/70. Climbing Stairs/70.py
class Solution(object):
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        if n<2:
            return 1
        a=[1]*n
        a[1]=2
        for i in range(2,n):
            a[i]=a[i-1]+a[i-2]
        return a[n-1]
